.github/workflows/scheduled.yaml

```diff
@@ -0,0 +1,49 @@
+# Tests in this workflow are run periodically.
+# They are generally very expensive and not on the critical path
+# for development
+# scheduled workflows only run in the default branch
+
+name: Scheduled Tests
+
+on:
+  schedule:
+    - cron: "42 3 * * 0"
```

Comment:
It feels trivial and unnecessary. To make a parallel, it's like requesting to add the comment

```
// sum the arguments. May overflow
int add (int n1, int n2) {
  return n1+n1;
}
```

If you demand it, I can add "run weekly" as a comment, but it obviously shares the same problem, although at least it's not as trivial

Reply:
> It feels trivial and unnecessary

I am surprised to see such an argument after a failure to describe what these numbers actually mean! IMHO, that failure proves that interpreting this combination of numbers is not trivial and that documenting this combination of numbers may be necessary (or at least a good idea) in this context, but I do not have the energy to insist on adding that comment.

> If you demand it, I can add "run weekly" as a comment

I do not like (and am not really in a position) to _demand_ anything. Please avoid using that word in relation to friendly change requests, especially requests supported by written reasonable arguments.

Adding a "once a week" comment would be better than adding nothing IMO, but I believe my suggestion is better than "weekly" or "once a week".

Reply:
Adding "once a week".

There is nothing mysterious about these numbers: the name of the section "cron" is a very obvious reference to the Unix recurrent task scheduler, it's been in use for just about 50 years. In fact, these numbers are about as self documenting as they come

Reply:
> There is nothing mysterious about these numbers

Yes, they are not mysterious. They are just difficult for humans to interpret _correctly_.

> these numbers are about as self documenting as they come

I strongly disagree because I do not equate existence of external documentation with self-documentation, especially in the context of cron configuration lines!

Reply:
> > runs in the night between Sunday and Monday (UTC)
> 
> That is not how I interpret POSIX cron [syntax](https://pubs.opengroup.org/onlinepubs/9699919799/utilities/crontab.html#tag_20_25_07) [used](https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/events-that-trigger-workflows#schedule) by GitHub Actions. I quoted a decyphering table at the end of this message for convenience.
> 
> My interpretation is that this job runs every Sunday at 3:42am (i.e. the night between Saturday and Sunday, not "the night between Sunday and Monday").

Doh, you're right. I was off by one (day)